### ISSUE TITLE
[REQUEST] moar metadata

### ISSUE THREAD

**Issue by xxxserxxx** [delucks/go-subsonic#9]
i'd like to start switching stmps over to this library; out of the box we'll need two OpenSubsonic API calls that are missing: `GetAlbumArt` and `GetLyricsBySongId`. Both are supported by gonic, and the first is supported by Navidrome; both are part of the OpenSubsonic API.  stmps has forks (with pending PRs) that use both, so we'd need them before switching.

Cheers

**Comment by dweymouth** [comment]
@xxxserxxx I don't think this repo is maintained anymore. I have a maintained fork at [supersonic-app/go-subsonic](https://github.com/supersonic-app/go-subsonic) that has support for `GetLyricsBySongID` and other (but not all, yet) OpenSubsonic extensions. Feel free to use it and I'd love PRs to help maintain it as well! (I also don't think `GetAlbumArt` exists as an OpenSubsonic endpoint?)
